src/script/api/script_event_types.hpp

```diff
@@ -799,7 +799,7 @@ class ScriptEventTownFounded : public ScriptEvent {
 
 /**
  * Event AircraftDestTooFar, indicating the next destination of an aircraft is too far away.
- * This event can be trigger when the current oder of an aircraft changes, usually either when
+ * This event can be trigger when the current order of an aircraft changes, usually either when
```

Comment:
"This event can be **trigger** when..." should be "This event should be **triggered** when...".

Reply:
Good spot. Thanks

Reply:
I've no idea what's going on here but it's very weird to see such an uncertainty in API description. Is this really such a random event that it's not guaranteed to trigger or the person writing this just wasn't sure? xD

Reply:
I would rewrite the sentence as "This event is triggered when the current order of an aircraft changes, **and the destination of the new current order is too far away compared to the aircraft's range**.

If the sentence here is kept as is, then there is one more typo: "usually either when loading is done or when **switch** manually" should be "usually either when loading is done or when **switched** manually" or "usually either when loading is done or when **changed** manually"

Reply:
I just decided to change it to "switched". I wasn't planning on changing the contents of the comments. Thanks anyway.